### DISCUSSION TITLE
حلِ یک مشکل

### DISCUSSION THREAD

**Discussion by ghost** [softwaretalks/newsletter#271] [Q&A]
<div dir="rtl">
من هر وقت می‌خوام Pull Request جدید ثبت کنم، بعد از زدنِ دکمه‌ی Fetch Stream این پیام رو می‌بینم:

`This branch is X commit ahead of softwaretalks:main
`

این در حالیه که همه‌ی کامیت‌ها merge شده. بنابراین هربار که می‌خوام تغییری بدم مجبورم فورکِ قبلی رو حذف کنم و دوباره از مخزنِ اصلی فورکِ جدید بگیرم. قطعاً راه‌حلش در استک هست؛ ولی چون نمی‌دونم چی سرچ کنم گفتم اینجا بپرسم شاید شما تجربه داشته باشید. 

یعنی انتظار دارم هربار که Fetch Stream می‌زنم این پیام رو ببینم که بتونم با خیالِ‌ راحت و بدونِ ساختِ فورکِ جدید Pull Request ثبت کنم:
`this branch is even with master
`</div>

**Comment by ohmydevops** [comment]
سلام ایوب عزیز امیدوارم هر جا هستی حالت خوب باشه
من خودم هم بلد نیستم. اما اگر راه‌حل رو یافتی خوشحال میشم اینجا به من هم یاد بدی یا منابع خوب رو معرفی کنی.

**Reply by ohmydevops** [reply]
یا بهتر این که به صورت یک مقاله در خود خبرنامه منتشر کنی :smile:

**Comment by ohmydevops** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
